src/main/java/net/earthcomputer/clientcommands/features/PlayerRandCracker.java

```diff
@@ -267,9 +287,10 @@ public static boolean throwItemsUntil(Predicate<Random> condition, int max) {
                 seed = (seed * MULTIPLIER + ADDEND) & MASK;
             rand.setSeed(seed ^ MULTIPLIER);
         }
-        if (itemsNeeded > max)
+        if (itemsNeeded > max) {
+            sendError(new LiteralText("Condition not possible or would require more than " + max + " item throws"));
```

Comment:
Don't put an error message in here, it can be called for reasons other than user interaction. If you want an error message, check the return value.